qa/integration/specs/command_line_spec.rb

```diff
@@ -36,4 +36,15 @@
     expect(execute.stderr_and_stdout).to include('--pipeline.id ID')
   end
 
+  it "starts without unexected warnings" do
+    execute = @logstash.run
+    lines = execute.stderr_and_stdout.split("\n")
+    expect(lines.shift).to match(/^(Using system java)|(Using bundled JDK)|(Using LS_JAVA_HOME defined java):/)
+    next_line = lines.shift
+    loop do
+      break unless next_line.match(/OpenJDK 64-Bit Server VM warning: Option UseConcMarkSweepGC was deprecated|warning: ignoring JAVA_TOOL_OPTIONS/)
+      next_line = lines.shift
+    end
+    expect(next_line).to match(/^Sending Logstash logs to/)
```

Comment:
Maybe the following could be more straightforward in communicating the intention to be a "consume lines loop"

```suggestion
    while (up_line = lines.shift).match(/OpenJDK 64-Bit Server VM warning: Option UseConcMarkSweepGC was deprecated|warning: ignoring JAVA_TOOL_OPTIONS/) do end
    expect(up_line).to match(/^Sending Logstash logs to/)
```

Reply:
Nice, I like that. Committed